lightning/src/util/config.rs

```diff
@@ -269,6 +269,13 @@ pub struct ChannelConfig {
 	/// [`Normal`]: crate::chain::chaininterface::ConfirmationTarget::Normal
 	/// [`Background`]: crate::chain::chaininterface::ConfirmationTarget::Background
 	pub force_close_avoidance_max_fee_satoshis: u64,
+	/// Sets how many the percent of the channel value the holder_max_htlc_value_in_flight_msat
+	/// will be assigned as.
+	///
+	/// Should be set to a value between 1-100, where the value signals the percantage to be used.
```

Comment:
What happens if the value is *not* within that range? Do we panic? Do we simply bound it to that range, etc.